### PR TITLE
Fix issue #65

### PR DESCRIPTION
**Pull request by henryyang42**
### Fix issue #65 and some enchancements
- set `is_active = True` using `createsuperuser`
- move send activation email function to `utils`
- fix typo `reciver` to `receiver`
- enable email with html

### CODE REVIEW

File: index/templates/index/activation_email.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Hey <b>{{username}}</b>, thanks for signing up.<br>
```
Comment (nickelcheng) [inline index/templates/index/activation_email.html]: Please add license.

File: users/admin.py [hunk 2]
```diff
@@ -64,10 +64,10 @@ def save(self, commit=True):
         user.set_password(self.cleaned_data["password1"])
         user.email = self.cleaned_data["email"]
         if commit:
-            user.is_active = False # not active until he opens activation link
```
Comment (nickelcheng) [inline users/admin.py]: Will a new-created user be set inactive if this line is removed?

File: utils/user_info.py [hunk 3]
```diff
@@ -146,3 +157,29 @@ def get_user_statistics(user):
...
+    new_profile.save()
+
+    # Send email with activation key
+    activation_link = request.META['HTTP_HOST'] + \
+        reverse('users:confirm', kwargs={'activation_key': activation_key})
+    email_subject = 'Account confirmation'
+    email_body = render_to_string('index/activation_email.html',
+                    {'username': username, 'activation_link': activation_link})
+    print email_body
+    msg = EmailMultiAlternatives(email_subject, email_body, 'nthucsoj@gmail.com', [email])
```
Comment (nickelcheng) [inline utils/user_info.py]: Please don't hardcode sender email, use config file instead

File: users/admin.py (hunk 2 above)
Comment (henryyang42) [inline users/admin.py]: In model we set ``is_active = models.BooleanField(default=False)``.
I think that line is not necessary

File: index/templates/index/activation_email.html (hunk 1 above)
Comment (henryyang42) [inline index/templates/index/activation_email.html]: Added

File: utils/user_info.py (hunk 3 above)
Comment (henryyang42) [inline utils/user_info.py]: I use ``emailInfo.py`` created by @drowsy810301 now.

File: users/admin.py (hunk 2 above)
Comment (nickelcheng) [inline users/admin.py]: ok, thanks.
BTW, the comment above this function is quite confusing, can you remove or modify it?

**Comment by nickelcheng** [comment]
This PR is ok for me. @CHIAHSINHOU please test this.

**Comment by CHIAHSINHOU** [comment]
This PR is ok for me.

File: utils/user_info.py [hunk 4]
```diff
@@ -146,3 +158,28 @@ def get_user_statistics(user):
...
+    email_subject = 'Account confirmation'
+    email_body = render_to_string('index/activation_email.html',
+                    {'username': username, 'activation_link': activation_link})
+    msg = EmailMultiAlternatives(email_subject, email_body, EMAIL_HOST_USER, [email])
+    msg.attach_alternative(email_body, "text/html")
+
+    try:
+        Thread(target=msg.send, args=()).start()
+    except:
+         logger.warning('There is an error when sending email to %s\' mailbox' % username)
```
Comment (bruce3557) [inline utils/user_info.py]: How about change to  ```logger.warning("There is an error when sending email to %s' mailbox"% username)```

**Comment by bruce3557** [comment]
Fine with this PR except the comment

File: utils/user_info.py (hunk 4 above)
Comment (henryyang42) [inline utils/user_info.py]: Changed

**Comment by nickelcheng** [comment]
going to merge.

**Pull request merged**